src/compounds/legacy-product-table/src/index.tsx

```diff
@@ -557,6 +557,45 @@ const Eligibility: React.FC<EligibilityProps> = ({
   )
 }
 
+interface TelephoneInfoProps {
+  telephone?: string
+}
+
+const TelephoneInfo: React.FC<TelephoneInfoProps> = ({ telephone }) => {
+  const {
+    theme: { name, colors = {} }
+  }: any = useThemeUI()
+
+  const iconColor = name === 'Money' ? colors.fuschia : colors.primary
+
+  return (
+    <div
+      sx={{
+        mr: '10px',
+        fontWeight: 600,
+        fontFamily: 'Varela Round,Open Sans,Helvetica Neue,Arial,sans-serif',
+        verticalAlign: 'middle',
+        display: 'flex',
+        height: '24px',
+        lineHeight: '15px',
+        textAlign: 'right'
+      }}
+    >
+      <span sx={{ mr: '5px', stroke: iconColor }}>
+        <Icon glyph="phone" color={iconColor} size={24} />
+      </span>
+      <span
+        sx={{
+          color: colors.text,
+          padding: '4px'
```

Comment:
is there a matching theme value

Reply:
I think I can use `theme.space.xxxs`, does it make sense?